index.html

```diff
@@ -183,5 +183,33 @@ <h3>Skills</h3>
             </li>
         </ul>
     </section>
+    <section class="contact-section">
+        <div class="contact-title">
+            <h2>Contact me</h2>
+        </div>
+        <div class="contact-text">
+            <span>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                I’d love to help with it!</span>
+        </div>
+        <form action="https://formspree.io/f/mdovrwgl" method="post">
+            <ul>
+                <li class="input-box">
+                    <label for="name"></label>
+                    <input type="text" name="user_name" id="name" placeholder="Name">
+                </li>
+                <li class="input-box"> 
+                    <label for="mail"></label>
+                    <input type="mail" name="user_name" id="mail" placeholder="example@mail.com">
+                </li>
+                <li class="input-box">
+                    <label for="msg"></label>
+                    <textarea name="user_message" id="msg" cols="30" rows="10" placeholder="Write your message here..."></textarea>
+                </li>
+                <li class="button">
+                    <button>Get in touch</button>
+                </li>
+            </ul>
+        </form>
```

Comment:
Greeting!
Great work on implementing this section, but there are a few things that you still needs to work on before moving to the next steps.

- Kindly, I suggest you use a `p` tag instead of `span` for good semantics and best practices. 😊 
- It seems like you forget to close the `input` tag, kindly I suggest you close the `input` tag as follow `<input  />`
## According to the project requirement:
 - I suggest you make sure all `input` fields have the `required` attribute to activate the validation on submit.
 - kindly make sure the name field has `maxlength="30"`
 - Kindly make sure the `message input` has `maxlength="500"` attribute.